src/components/molecules/MarketStats.tsx

```diff
@@ -19,9 +22,30 @@ interface MarketStatsResponse {
 
 const refreshInterval = 60000 // 60 sec.
 
+const getTotalPoolsValues = gql`
+  query PoolsData {
+    poolFactories {
+      totalLockedValue
+      totalLiquidity
+      totalSwapVolume
+      totalSwapFee
+      poolCount
+      finalizedPoolCount
+    }
+  }
+`
+
 export default function MarketStats(): ReactElement {
   const [ref, inView] = useInView()
   const [stats, setStats] = useState<MarketStatsResponse>()
+  const [totalLockedValue, setTotalLockedValue] = useState()
```

Comment:
when no initial state is set, the state should be explicitly typed depending on what `totalLockedValue` should be. Think in this case it would be `useState<string>()`